src/main/java/io/cloudsoft/winrm4j/winrm/WinRmTool.java

```diff
@@ -24,13 +24,49 @@ public static WinRmTool connect(String address, String username, String password
         return tool;
     }
 
+    /**
+     * Execute a list of Windows Native commands as one command.
+     * The method translates the list of commands to a single String command with a <code>"\r\n"</code> delimiter and a terminating one.
+     * It is creating a new Shell on the destination host each time it is being called.
```

Comment:
This kind of thing is implementation details rather than part of the contract. I'd therefore be very hesitant about including it in the javadoc.